tutorials/NWB Tutorials/read_in_NWB.m

```diff
@@ -0,0 +1,51 @@
+%% *MatNWB Setup*
+% Start by setting up your MATLAB workspace. The code below clones the
+% MatNWB repo to the current directory and adds the folder ontaining the 
+% MatNWB package to the MATLAB search path. MatNWB works by automatically 
+% creating API classes based on a defined schema. Running the generateCore() function
+% generates these classes for the lastest schema version.
+
+!git clone https://github.com/NeurodataWithoutBorders/matnwb.git
+cd ../matnwb
+addpath(genpath(pwd));
+generateCore();
+%% *Load in Image Files*
+% The extractor.m function function requires two inputs
+
+%  1) An image stack with dimensions [rows, columns, frames]
+%  2) A config strcuture with the parameters for the EXTRACT algorithm
+
+
+% You can read in an nwb file with the nwbRead function from MatNWB. The code 
+% below reads in a file downloaded from this DANDIset <https://gui.dandiarchive.org/#/dandiset/000054 
```

Comment:
```suggestion
% below reads in a file downloaded from this DANDIset https://dandiarchive.org/dandiset/000054/0.210819.1547 
```

@cechava, Satra is pretty adamant that linking to dandisets be done with the URL that you get when you click "Share" on the dandiset landing page, rather than just copy/pasting the URL from the address bad.